ksql-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsLocator.java

```diff
@@ -85,17 +86,26 @@
 
     final HostInfo activeHost = metadata.getActiveHost();
     final Set<HostInfo> standByHosts = metadata.getStandbyHosts();
-    LOG.debug("Before filtering: Active host {} , standby hosts {}", activeHost, standByHosts);
 
-    final List<KsqlHostInfo> allHosts = Stream.concat(Stream.of(activeHost), standByHosts.stream())
-        .map(this::asKsqlHost)
-        .collect(Collectors.toList());
+    // If the lookup is for a forwarded request, only filter localhost
+    List<KsqlHostInfo> allHosts = null;
+    if (routingOptions.skipForwardRequest()) {
+      LOG.debug("Before filtering: Local host {} ", localHost);
```

Comment:
just 1 debug statement after the if block, that print the flag value, and `allHosts`?

Reply:
If I do that, I will loose the information of who is active and who is standby